src/displayapp/screens/Dice.h

```diff
@@ -1,12 +1,13 @@
 #pragma once
 
+#include "components/motion/MotionController.h"
 #include "displayapp/screens/Screen.h"
 #include "displayapp/widgets/Counter.h"
+#include "lvgl/lvgl.h"
```

Comment:
sorry about the misunderstanding, but lvgl isn't "our" project, it is in our project dir through a submodule, but it is not ours. So I'd move the include down and use  `<>`

Reply:
ah yes-- sorry about that, thanks for explaining! fixed in 4647f8aa3e3f464f93e3cdd877d23daecaace796